File: 1748.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        unordered_map<int, int> count;
        int res = 0;

        for(int i = 0; i < nums.size(); i++) {
            count[nums[i]]++;
        }

        for(auto it = count.begin(); it != count.end(); it++) {
            if(it->second == 1) {
                res += it->first;
            }
        }
        return res;
    }
};

File: 1748.py
from collections import defaultdict

class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        count = defaultdict(lambda:0)
        res = 0

        for num in nums:
            count[num] += 1
        
        for key, value in count.items():
            if value == 1:
                res += key

        return res